cipher.go

```diff
@@ -46,8 +47,10 @@ func Cipher(payload []byte, mask [4]byte, offset int) {
 	// Get number of uint64 parts remaining to process.
 	n = (n - ln - rn) >> 3
 	for i := 0; i < n; i++ {
-		v := (*uint64)(unsafe.Pointer(&payload[ln+(i<<3)]))
-		*v = *v ^ m2
+		idx := ln + (i << 3)
+		p := binary.LittleEndian.Uint64(payload[idx : idx+8])
+		p = p ^ m2
+		binary.LittleEndian.PutUint64(payload[idx:idx+8], p)
```

Comment:
Actually this is a little bit slower than we had, but it is the best solution for now I think. Thank you!